app/controllers/admin/api/account/proxy_configs_controller.rb

```diff
@@ -25,6 +25,8 @@ class Admin::Api::Account::ProxyConfigsController < Admin::Api::BaseController
   #
   ##~ op.parameters.add @parameter_access_token
   ##~ op.parameters.add @parameter_environment
+  ##~ op.parameters.add :name => "host",    :description => "Find by host",    :dataType => "string", :required => false, :paramType => "query"
+  ##~ op.parameters.add :name => "version", :description => "Find by version", :dataType => "string", :required => false, :paramType => "query"
```

Comment:
Should we describe them "Filter by..." instead of "Find by..."?

Reply:
Makes sense 👍🏼 In the following PR I will add this improvement 👍🏼